### PR TITLE
Python code for ESGain O2O Deployment 

### PR DESCRIPTION
**Pull request by ggovi**
#### PR description:
- Clean up of python db credential management
- Adaptation of existing O2O code for the above changes
- Integration of deployment-related python code for ESGain O2O

#### PR validation:

Unit-tests + validation in online environment

### CODE REVIEW

**Comment by ggovi** [comment]
please test

File: CondTools/Ecal/python/get_ES_gain_from_tag.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+db_target_account = 'CMS_ES_CONF'
+db_service,db_user,db_pwd = auth.get_db_credentials( db_target_account )
+conn_str = u'%s/%s@%s' %(db_user,db_pwd,db_service)
+def main(argv):
+  #print "Script to extract the gain used by ES providing the TAG of the run key"
+  #print "Launch it passing as inputs:
+  # - the tag name
+ 
+  if len(argv) !=2:
+    print "Wrong number of parameters"
```
Comment (davidlange6) [inline CondTools/Ecal/python/get_ES_gain_from_tag.py]: please make sure this script is python3 compatible.

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**